Packs/Base/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Base",
     "description": "The base pack for Cortex XSOAR.",
     "support": "xsoar",
-    "currentVersion": "1.16.6",
+    "currentVersion": "1.16.7",
```

Comment:
I think you can compliment yourself with a minor release.

Reply:
Will do. :-)